diff -Naur a/src/Makefile.am b/src/Makefile.am
--- a/src/Makefile.am	2014-03-26 21:40:47.000000000 +0000
+++ b/src/Makefile.am	2014-04-09 14:55:38.784712385 +0100
@@ -48,7 +48,7 @@
 if LINUX
 giada_LDADD = -lsndfile -lfltk -lXext -lX11 -lXft -lXpm -lm  \
 							rtaudio/librtaudio.a -ljack -lasound -lpthread -ldl \
-							-lpulse-simple -lpulse -lsamplerate -lrtmidi
+							-lsamplerate -lrtmidi
 endif
 if WINDOWS
 giada_LDADD   = -lole32 rtaudio/librtaudio.a -ldsound -lwinmm -lwsock32 \
@@ -81,7 +81,7 @@
 libs:
 if LINUX
 	@cd rtaudio; echo "Building RtAudio for Linux..."; \
-	./configure --with-jack --with-alsa --with-pulse; \
+	./configure --with-jack --with-alsa; \
 	make;
 endif
 if WINDOWS
diff -Naur a/src/Makefile.in b/src/Makefile.in
--- a/src/Makefile.in	2014-03-26 21:40:53.000000000 +0000
+++ b/src/Makefile.in	2014-04-09 14:55:54.941712427 +0100
@@ -252,7 +252,7 @@
 # inside configure.ac
 @LINUX_TRUE@giada_LDADD = -lsndfile -lfltk -lXext -lX11 -lXft -lXpm -lm  \
 @LINUX_TRUE@							rtaudio/librtaudio.a -ljack -lasound -lpthread -ldl \
-@LINUX_TRUE@							-lpulse-simple -lpulse -lsamplerate -lrtmidi
+@LINUX_TRUE@							-lsamplerate -lrtmidi
 
 @OSX_TRUE@giada_LDADD = /usr/local/lib/libsndfile.a -lm -lpthread -lfltk -lrtmidi \
 @OSX_TRUE@                rtaudio/librtaudio.a
@@ -626,7 +626,7 @@
 
 libs:
 @LINUX_TRUE@	@cd rtaudio; echo "Building RtAudio for Linux..."; \
-@LINUX_TRUE@	./configure --with-jack --with-alsa --with-pulse; \
+@LINUX_TRUE@	./configure --with-jack --with-alsa; \
 @LINUX_TRUE@	make;
 @WINDOWS_TRUE@	@cd rtaudio; echo "Building RtAudio for Windows..."; \
 @WINDOWS_TRUE@	./configure --with-asio --with-ds; \
