diff -Naur a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2014-09-25 12:20:34.000000000 +0100
+++ b/CMakeLists.txt	2016-07-28 05:10:44.137380730 +0100
@@ -1,5 +1,9 @@
 project(Kdenlive)
-
+if(COMMAND cmake_policy)
+    cmake_policy(SET CMP0000 OLD)
+    cmake_policy(SET CMP0003 OLD)
+    cmake_policy(SET CMP0059 OLD)
+endif(COMMAND cmake_policy)
 # An odd patch version number means development version, while an even one means
 # stable release. An additional number can be used for bugfix-only releases.
 set(KDENLIVE_VERSION 0.9.10)
